Rakefile

```diff
@@ -37,7 +37,7 @@ end
 
 desc "Run RSpec tests that require slow to start puppet containers"
 RSpec::Core::RakeTask.new(:puppetserver) do |t|
-  t.rspec_opts = '--tag puppetserver --tag puppetdb --tag omi'
+  t.rspec_opts = '--tag puppetserver --tag puppetdb --tag omi --tag kerberos'
```

Comment:
I think this also needs `--tag ~kerberos` added to the `unit`, `travisci`, and `appveyor` rake tasks

Reply:
Thank you - good catch! Totally missed in the logs that those tests were running multiple times in Travis. Updating!

Reply:
And now that AppVeyor is running again, on the repush we'll get validation that tests aren't running in that environment either.